packages/common/src/utils.ts

```diff
@@ -397,3 +397,39 @@ export function intToBigInt(value: IntegerType, signed: boolean): bigint {
 export function with0x(value: string): string {
   return !value.startsWith('0x') ? `0x${value}` : value;
 }
+
+export function hexToBigInt(hex: string): bigint {
+  if (typeof hex !== 'string')
+    throw new TypeError('hexToNumber: expected string, got ' + typeof hex);
+  // Big Endian
+  return BigInt(`0x${hex}`);
+}
+
+export function hexToBytes(hex: string): Uint8Array {
+  if (typeof hex !== 'string')
+    throw new TypeError('hexToBytes: expected string, got ' + typeof hex);
+  if (hex.length % 2)
+    throw new Error(`hexToBytes: received invalid unpadded hex, got: ${hex.length}`);
+  const array = new Uint8Array(hex.length / 2);
+  for (let i = 0; i < array.length; i++) {
+    const j = i * 2;
+    array[i] = Number.parseInt(hex.slice(j, j + 2), 16);
+  }
+  return array;
+}
```

Comment:
What happens if this method is supplied a `0x` prefixed string? What happens if it's supplied a non-hex string?